codebuild/bin/install_al2_dependencies.sh

```diff
@@ -16,6 +16,11 @@
 set -eu
 source ./codebuild/bin/s2n_setup_env.sh
 
+if [[ ${DISTRO} != "amazon linux" ]]; then
+    echo "Target AL2, but running on $DISTRO: Nothing to do."
+    exit 0
```

Comment:
Is this nothing to do, or an error?

Reply:
Nothing todo; original plan was to call all the install_*_dependencies and let them decide- thought that is not part of this PR.